app/routes/bestuurseenheid/zitting.js

```diff
@@ -1,21 +1,10 @@
 import Route from '@ember/routing/route';
-import { format } from 'date-fns';
-import { nl } from 'date-fns/locale';
+import { inject as service } from '@ember/service';
 
 export default class BestuurseenheidZittingRoute extends Route {
-  breadCrumb = {};
+  @service store;
 
   model(params) {
-    return this.store.findRecord('zitting', params.id);
-  }
-
-  async afterModel(zitting) {
-    const bestuursorgaan = await zitting.bestuursorgaan;
-    const bestuursorgaanInTijd = await bestuursorgaan.isTijdsspecialisatieVan;
```

Comment:
I think these relationships need to be loaded again. It was one of the open remarks on the breadcrumds refactor PR: https://github.com/lblod/frontend-gelinkt-notuleren-publicatie/pull/21#discussion_r636796460

I would just use includes when loading the record in the model hook though.

Reply:
This is something I have noted, but didn't think to fix. This is not as difficult as I imagined. I'm not setting the breadCrumb here though, but on the template. Loading these relations in the afterModel should also be good for FastBoot, right?

Reply:
I tried replacing it with include on the model hook, but that causes a time-out:
```javascript
model(params) {
  return this.store.findRecord('zitting', params.zitting_id, {
    include: 'bestuursorgaan,bestuursorgaan.isTijdsspecialisatieVan',
  });
} 
```
so, using afterModel and awaits instead.

Reply:
To clarify, the include needed to be written as:
```javascript
...
include: 'bestuursorgaan,bestuursorgaan.is-tijdsspecialisatie-van',
...
```
and that works as intended.

Reply:
FYI, this can be shortened to `'bestuursorgaan.is-tijdsspecialisatie-van'` since that will also include the `bestuursorgaan` records.